bigquery/tables.rb

```diff
@@ -53,10 +51,9 @@ def delete_table project_id:, dataset_id:, table_id:
   # dataset_id = "ID of the dataset delete table from"
   # table_id   = "ID of the table to delete"
 
-  require "google/cloud"
+  require "google/cloud/bigquery"
 
-  gcloud   = Google::Cloud.new project_id
-  bigquery = gcloud.bigquery
+  bigquery = Google::Cloud::Bigquery.new
```

Comment:
project: project_id is necessary because this is inside a function which has a project_id parameter.